api/envoy/service/cluster/v2alpha/event_reporting_service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package envoy.service.cluster.v2alpha;
```

Comment:
I think this should have its own tree in the `envoy.service` hierarchy, e.g. `envoy.service.event_reporting.v2alpha`.

Reply:
yep, good idea